### ISSUE TITLE
Training Result Bad

### ISSUE THREAD

**Issue by HuaDeity** [Coordi777/Accelerating-Diffusion-for-SAR-to-Optical-Image-Translation#2]
Hello,

I have attempted to replicate your model but have encountered poor results. I am reaching out for assistance with the following queries:

1. In your paper, you mentioned selecting data from the original SEN12 dataset. Could you provide details on the selection process?
2. Does the file `configs_example.yaml` reflect the settings you used? Are there specific arguments I should be aware of?
3. In the `train.py` file, there is an argument `pretrained_model_path` which appears to be related to the scheduler. Could you clarify which pretrained model was used here? I attempted to use the scheduler from Stable Diffusion 1.5, but the results were unsatisfactory.

**Comment by Mizar29** [comment]
Hello! I also encountered the same problem about pretrained_model_path. Have you solved it now?

**Comment by HuaDeity** [comment]
@Mizar29 No, since there was no response from the author, I’ve stopped trying.

**Comment by Mizar29** [comment]
@HuaDeity Thank you for your reply! In addition, you mentioned that the training results were not good. How many epochs did you train?
